gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# Assign additional (advertised) addresses to the local device,
+# add a local/remote subflow for each address
+
+usage() { echo "$0 [-e <endpoints>] [-m <signal|subflow>] [-b]" 1>&2; exit 1; }
+
+create_endpoints() {
+    local host= flags= ep=1 max=$1 hn=$2 nn=$3 pl=$4 sig=${5:-0} bck=${6:-0}
```

Comment:
(a bit difficult to understand with so short names, I had to look at how you were using it below :) )

PS: you can also write `host` instead of `host=` :)

Reply:
> (a bit difficult to understand with so short names, I had to look at how you were using it below :) )
I'll add a comment in the line abbove, that explains the function arguments. Ok?
> PS: you can also write `host` instead of `host=` :)
... right. Oh well, there were buggy shells in the past that did weird things if we didn't assign the empty string to local variables.  But modern bash can live also without it :)